hack/community/generate.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Generate the new community manifests, from the current WMCO manifests.
+
+# Must be ran within WMCO root.
+
+# Example:
+# Cut community v5.1.0 for community-4.10
+# Run: bash ./hack/community/generate.sh 5.1.0 community-4.10
+
+clone_community() {
+  local CO_GIT="https://github.com/${CO_REPO}.git"
+
+  git clone "${CO_FORK}" "${CO_DIR}"
+  pushd "${CO_DIR}"
+  git remote add upstream "${CO_GIT}"
+  git fetch upstream main:upstream/main
+  git checkout upstream/main
+  git config user.name "${GITHUB_NAME}"
+  git config user.email "${GITHUB_EMAIL}"
+  echo "   ${CO_PROJECT} cloned"
+  popd
+}
+
+replace() {
+    CREATED_AT=$(date +"%Y-%m-%dT%H:%M:%SZ")
+    VALUE="community-windows-machine-config-operator"
+    NAME="${VALUE}-${OPERATOR_VERSION}"
+    DISPLAY_NAME="Community Windows Machine Config Operator"
+    MATURITY="preview"
+    QUAY_IMAGE=
+    DESCRIPTION=$(cat hack/community/csv/description.md)
+
+    /tmp/yq eval --exit-status --inplace '
+      .metadata.annotations.createdAt |= strenv(CREATED_AT) |
+      .metadata.name |= strenv(NAME) |
+      .spec.description |= strenv(DESCRIPTION) |
+      .spec.displayName |= strenv(DISPLAY_NAME) |
+      .spec.install.spec.deployments[0].spec.template.spec.containers[0].env[2].value |= strenv(VALUE) |
+      .spec.install.spec.deployments[0].spec.template.spec.containers[0].image |= strenv(QUAY_IMAGE) |
+      .spec.maturity |= strenv(MATURITY) |
+      .spec.version |= strenv(OPERATOR_VERSION)
+    ' "${CO_CSV}"
+}
+
+generate_manifests() {
+
+  echo "Update operator manifests"
+  cp -r "${OPERATOR_MANIFESTS}" "${CO_OPERATOR_DIR}/${OPERATOR_VERSION}"
+  pushd "${CO_DIR}"
+  git checkout -B "${OPERATOR_VERSION}"
+  CO_CSV="${CO_OPERATOR_DIR}/${OPERATOR_VERSION}/${CSV}"
+
+  replace "$CO_CSV"
+}
+
+submit_PR() {
+  echo "Submit PR to community operators"
+  echo "commit changes"
+  git add --all
+  git commit -s -m "community-windows-machine-config-operator.v${OPERATOR_VERSION}"
+  git push --set-upstream --force origin HEAD
+
+  echo "submit PR"
+  # Create PR
+  curl "https://api.github.com/repos/${CO_REPO}/pulls" --user "${GITHUB_USER}:${GITHUB_TOKEN}" -X POST \
+     --data '{"title": "'"$(git log -1 --format=%s)"'", "base": "main", "body": "An automated PR to update
+      community-windows-machine-config-operator to v'"${OPERATOR_VERSION}"'", "head":
+      "'"${GITHUB_USER}:${OPERATOR_VERSION}"'"}'
+  popd
+
+  echo "Done"
+}
+
+OPERATOR_VERSION="$1"
+OPERATOR_MANIFESTS="bundle/manifests"
```

Comment:
In addition, consider capturing the second argument as well.

```suggestion
OPERATOR_VERSION="$1"
COMMUNITY_VERSION="$2"
OPERATOR_MANIFESTS="bundle/manifests"
```